CONTRIBUTING.md

```diff
@@ -1,5 +1,39 @@
 # Contributing
 
-Thanks for your interest in contributing to Bearer.
+Thanks for your interest in contributing to Bearer Rules.
 
-You can find guides and everything you need to started in the [Contributing to Bearer](https://docs.bearer.com/contributing/) documentation.
+## Run the tests
+
+Our integration tests run against the latest binary release of Bearer. These tests are included in the CI process, so you shouldn't need to run them in their entirety (which is good because they take a long time to run locally).
+
+The more realistic case is to run tests for a single rule. You can do this locally from a bash shell with the following command:
+
+```bash
+  ./scripts/run-tests.sh path-to-rule-testdata-dir
+```
+
+For example
+
+```bash
+  ./scripts/run_tests.sh ruby/lang/logger/
+```
+
+## Add a new rule to bearer-rules
+
+Here are the basic steps to adding a new rule.
+
+1. Create a rule YAML file following the guide [here](https://docs.bearer.com/guides/custom-rule/)
+2. Add a folder of test data. This includes example code that should (or for "ok" cases, should not) trigger your new rule
```

Comment:
I wonder if there's a good example folder to link to here? Maybe one with multiple OKs and multiple trigger cases. Probably something to expand on in an added section of the custom guide later.